src/VisualStudio/Core/Impl/Options/GridOptionPreviewControl.xaml

```diff
@@ -35,23 +35,22 @@
             <converters:MarginConverter x:Key="MarginConverter" />
         </Grid.Resources>
         <Grid.RowDefinitions>
-            <RowDefinition Height="Auto" />
-            <RowDefinition Height="*" />
-            <RowDefinition Height="5" />
-            <RowDefinition Height="*" />
+            <RowDefinition Height="Auto" MinHeight="74.857" />
+            <RowDefinition Height="115*" />
+            <RowDefinition Height="4.571" />
```

Comment:
these values seem wonky.  why do we have strange values like 74.857 instead of just "75".  ?

Reply:
I added the new button via the XAML designer in Visual Studio, which may have resulted in the precise numbers.

Reply:
we should still fix this up :)